src/kvirc/ui/KviMenuBar.cpp

```diff
@@ -174,6 +176,14 @@ void KviMenuBar::setupSettingsPopup(QMenu *pop)
 	QAction *pAction = opt->addAction(*(g_pIconManager->getSmallIcon(KviIconManager::Toolbar)),__tr2qs("Toolbars"));
 	pAction->setMenu(m_pToolbarsPopup);
 
+#ifndef COMPILE_ON_MAC
+	m_pMenuBarAction = opt->addAction(*(g_pIconManager->getSmallIcon(KviIconManager::StatusBar)),__tr2qs("Show Menu Bar"),m_pFrm,SLOT(toggleMenuBar()));
```

Comment:
The option is marked by default, like the one we already have for the status bar:

![2016-04-03_12-24-35](https://cloud.githubusercontent.com/assets/1001553/14231964/125971c8-f997-11e5-9162-d2e3e91d82c8.png)